README.md

```diff
@@ -32,16 +34,22 @@ $ cd git-machete
 $ python setup.py install
 ```
 
-To install you need to have Python with installed `pip` and `setuptools`. Recommended Python env is installed with [Pyenv](https://github.com/pyenv/pyenv).
+To enable bash completion, ensure that you have sourced files from `${HOME}/.local/etc/bash_completion.d/` in your bash completion config.
```

Comment:
I'd add sth like:

You can do it explicitly by adding the following line to your .bashrc:
```bash
. ~/.local/etc/bash_completion.d/git-machete-prompt
```